LandLord-api/src/main/java/biz/princeps/landlord/api/IMobManager.java

```diff
@@ -24,9 +24,10 @@
     IMob get(EntityType type);
 
     /**
-     * Gets a collection of all available Mobs.
+     * Gets a list of all available Mobs.
      *
-     * @return a collection of all mobs
+     * @return a list of all mobs
      */
-    Collection<IMob> values();
+    List<IMob> values();
```

Comment:
This is an API/ABI breaking change. Is it required? From what I've seen, it's not necessary to expose that Lists are used internally.

Reply:
Gonna revert this.